examples/node/pdf2png/README.md

```diff
@@ -0,0 +1,16 @@
+## Overview
+
+Example to demonstrate PDF.js library to convert PDF file to a PNG image.
```

Comment:
I would make this `Example to demonstrate converting a PDF file to a PNG image using the PDF.js library.`